bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcPanel.scala

```diff
@@ -47,6 +47,12 @@ sealed trait ItcPanel extends GridBagPanel {
   def display: Component
   def visibleFor(t: SPComponentType): Boolean
 
+  /** TODO: we need to display a list of asterism members and let the user select one. For now we
+    * just select the first target in the asterism.
+    */
+  def selectedTarget: Option[Target] =
+    Option(owner.getContextTargetEnv).map(_.getAsterism.allTargets.head)
+
```

Comment:
Yeah there are other places in code that I have avoided so far that look at the magnitude table. I don't know how to deal with these yet. It may be the case that it only happens in a context where we "know" there's only one target.